net/freeradius/Makefile

```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		freeradius
-PLUGIN_VERSION=		1.9.22
+PLUGIN_VERSION=		1.9.23
```

Comment:
we don't have to bump the version, we can reuse 1.9.22 as the changes have just been merged